src/index.d.ts

```diff
@@ -43,13 +43,14 @@ export type Ref<T> = RefObject<T> | RefCallback<T> | null;
 
 export type ComponentChild =
 	| VNode<any>
-	| object
 	| string
 	| number
+	| Iterable<ComponentChild>
```

Comment:
I don't think we support any iterables as component children. Can't find code in our source that explicitly invokes the iterator protocol.

Reply:
So do you think is better the Array type here?